packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -22,28 +18,16 @@ const WalletsMigrationFailed = observer(() => {
         setWalletsMigrationFailedPopup(false);
     };
 
-    const validateClickOutside = (e: MouseEvent) => {
-        return is_wallet_migration_failed && !wallets_migration_failed_ref?.current?.contains(e.target as Node);
-    };
-
-    useOnClickOutside(wallets_migration_failed_ref, handleClose, validateClickOutside);
-
     return (
-        <Modal
-            is_open={is_wallet_migration_failed}
-            toggleModal={handleClose}
-            width={is_mobile ? '32.3rem' : '44rem'}
-            should_header_stick_body={false}
-            has_close_icon={false}
-        >
-            <div ref={wallets_migration_failed_ref}>
-                <Modal.Body className={`wallets-migration-failed${mobile_add_class}`}>
+        <Modal is_open={is_wallet_migration_failed} toggleModal={handleClose} width={is_mobile ? '32.3rem' : '44rem'}>
+            <div>
+                <Modal.Body className='wallets-migration-failed'>
                     <Text
                         as='h1'
                         size={is_mobile ? 'xs' : 's'}
                         color='prominent'
                         weight='bold'
-                        className={`wallets-migration-failed__title${mobile_add_class}`}
```

Comment:
@sergei-deriv why did you remove the mobile class?

Reply:
@mahdiyeh-deriv it's because I've add `@include mobile` to scss file and we don't need a separate class for mobile